modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js

```diff
@@ -461,20 +459,14 @@ class LayoutProvider extends Component {
 
 	_handlePaginationNextClicked() {
 		const {activePage, pages} = this.state;
-
-		handlePaginationNextClicked(
-			{
-				activePage,
-				pages
-			},
-			this.dispatch.bind(this)
-		);
+		const pageIndex = Math.min(activePage + 1, pages.length - 1);
+		handlePaginationItemClicked({pageIndex}, this.dispatch.bind(this));
```

Comment:
Hey @carolmariaabb, have you tested this fix against IE 11? I'm afraid that the use of `{` and `}` in `{pageIndex}` can raise problems with this browser. Can you please check it? Thanks.

Reply:
Hi @natocesarrego , I just checked and it works fine in IE 11.
Thanks.

Reply:
Thanks, @carolmariaabb! 🙂